packages/webapp/pages/onboarding.tsx

```diff
@@ -392,7 +372,7 @@ export function OnboardPage(): ReactElement {
   }
 
   return (
-    <div className="z-3 flex h-full max-h-screen min-h-screen w-full flex-1 flex-col items-center overflow-x-hidden bg-[#0e1019]">
+    <div className="z-3 flex h-full max-h-screen min-h-screen w-full flex-1 flex-col items-center overflow-x-hidden">
```

Comment:
is the bg removal here intentional?

Reply:
Yep was only added as temp video match fix